src/tools/fuzzing.h

```diff
@@ -318,11 +318,8 @@ class TranslateToFuzzReader {
     TODO_SINGLE_COMPOUND(type);
     switch (type.getBasic()) {
       case Type::anyref:
-        assert(wasm.features.hasAnyref());
-        if (wasm.features.hasReferenceTypes()) {
-          options.push_back(Type::funcref);
-          options.push_back(Type::externref);
-        }
+        options.push_back(Type::funcref);
+        options.push_back(Type::externref);
```

Comment:
I'm not too opinionated about this, but the fuzzer still does not assume feature dependencies have been completely resolved; it checks that here and there. How about restoring this condition and maybe we can remove it when we add feature implication feature.

Reply:
That's exactly why I assumed that if `anyref` is present, the fuzzer wouldn't have emitted it earlier if reference types aren't enabled, hence removing the `assert`. Yet, `exnref` isn't a subtype if exception handling isn't enabled, so the fuzzer shouldn't emit it in this case, and I thought this change would be an improvement. May I ask for a concrete suggestion on what to change it to?

Reply:
Gone ahead and restored the condition. Hope it's correct now :)